src/main.ts

```diff
@@ -83,7 +85,7 @@ export async function run(): Promise<void> {
       `Workflow steps \n ${JSON.stringify(response.graph.steps, null, 2)}`
     )
     core.notice(
-      `You can find the visual representation of the steps graph on [Chassy Web Platform](https://console.test.chassy.dev/workflows/${response.workflowId}/${workflowExecutionId})`
+      `You can find the visual representation of the steps graph on [Chassy Web Platform](${frontendBaseUrl}/workflows/${response.workflowId}/${workflowExecutionId})`
```

Comment:
Please also change workflows/${response.workflowId}/${workflowExecutionId} to workflows/${response.workflowId}?runId=${workflowExecutionId}
workflowExecutionId was moved to query parameters

Reply:
updated